src/uwtools/cli.py

```diff
@@ -697,9 +709,12 @@ def _check_file_vs_format(file_arg: str, format_arg: str, args: Args) -> Args:
     return args
 
 
-def _check_quiet_vs_verbose(args) -> Args:
-    if args.get(STR.quiet) and args.get(STR.verbose):
-        _abort("Specify at most one of %s, %s" % (_switch(STR.quiet), _switch(STR.verbose)))
+def _check_verbosity(args) -> Args:
+    if sum([args.get(STR.debug, 0), args.get(STR.quiet, 0), args.get(STR.verbose, 0)]) > 1:
+        _abort(
+            "Specify at most one of %s, %s, or %s"
+            % (_switch(STR.debug), _switch(STR.quiet), _switch(STR.verbose))
+        )
```

Comment:
Do we want to penalize both a `debug` and a `verbose` flag? Maybe I'm just the crotchety debugger that would be annoyed by this. 😬

Reply:
Strictly it would be redundant verbose calls, but I can see what you mean

Reply:
I'm in favor of _not_ penalizing it. For example, if I have a git branch with unmerged commits and I say
```
git branch -d foo
```
git tells me that I have to use `-D` to force-delete it. Then I write
```
git branch -d foo -D
``
because I up-arrow to get the previous command, and I'm happy _every single time_ that I don't have to actually change `-d` but can just strengthen it with `-D`.